src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```diff
@@ -14,58 +14,57 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package reset implements factory reset for Windows.
-package reset
+package factoryreset
 
 import (
-	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path"
 	"strings"
-	"unsafe"
+	"syscall"
 
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/directories"
 	"github.com/sirupsen/logrus"
-	"golang.org/x/sys/windows"
+	"golang.org/x/text/encoding/unicode"
 )
 
-const (
-	appName = "rancher-desktop"
-)
+func PowershellKillWindows() {
```

Comment:
This function no longer has anything to do with PowerShell. (The function name shouldn't have referred to its implementation, but what it's meant to do.)